### DISCUSSION TITLE
Compensating contributors

### DISCUSSION THREAD

**Discussion by rviscomi** [HTTPArchive/almanac.httparchive.org#2790] [Ideas]
I've struggled with the idea of compensating contributors since day 1. I am so grateful that people choose to give up so much of their time to this project and I want to make sure that everyone feels valued and appreciated. In the 2021 contributor survey, this topic was raised by a contributor and so I wanted to open it up for a wider discussion to hear what other people think. A few questions to prompt the discussion:

- Do you feel like it's necessary to compensate contributors?
- Would you want all contributors to be compensated equally, regardless of role or amount of effort?
- Should contributors be compensated monetarily or with some other sort of "thank you" gift?
- How can we ensure that the process is fair?

Compensating contributors fairly is the hardest problem and I've been dreading it for the last three years. I only want to do what's fair and I wouldn't want anyone to feel undervalued, so the easiest thing is just to default to volunteer contributions.

That said I do have a couple of ideas. Please don't take this as something we will definitely do, it's more of a request for feedback.

The first idea is for HTTP Archive to raise money from [our corporate sponsors](https://httparchive.org/about#sponsors) to allocate towards printing and shipping every contributor a physical copy of the Web Almanac for each year they've contributed. Everyone regardless of their role/year would be eligible.

The second idea, which is not necessarily mutually exclusive with the first, is to crowdfund compensation for individual contributors. HTTP Archive has an account on [Open Collective](https://opencollective.com/httparchive) where we could create fundraising goals and contributors could request a goal be set up in their name and set the amount they're looking to raise. Members of the community (including corporate sponsors) could chip in to fund a particular contributor. At the end of the project, the contributors would receive however much they were able to raise.

What I like about the first idea is that everyone gets something of equal value. It's more sentimental than valuable though, and not really "compensation" so much as a token of thanks. Maybe that's ok.

What I like about the second idea is that everyone has equal access to crowdfund an amount they feel would be fair. It absolves me from the horrible position of having to decide how much any one person's contributions are worth. However it does turn into something of a popularity contest and contributors may be competing against each other for limited funds. It could get super messy.

Any other thoughts or ideas?

**Comment by roryhewitt** [comment]
I think #2 is just too likely to go horribly wrong - as you say, it could (would?) turn into a popularity contest and someone will end up unhappy. Perhaps there could be a general fund into which either Open Collective money or our corporate sponsors could contribute, with the money split evenly amongst everyone (or maybe different amounts for authors, analysts, reviewers etc.)...

For me (author 2020, reviewer 2021), I'd be happy with #1, but I recognize others may feel differently.

**Comment by NishuGoel** [comment]
I agree the #2 could end up becoming unfair. What could be done instead is:
1) a physical copy and 
2) recognition of sorts like below:

<img width="448" alt="Screenshot 2022-01-28 at 7 02 09 PM" src="https://user-images.githubusercontent.com/26349046/151556779-b2d3f13f-2fda-4722-888f-b6b15cb801cc.png">

**Reply by tunetheweb** [reply]
Do you mean Author's name? Or the words "Author"?

Individualised copies can add significantly to cost (bulk orders of same thing are a lot cheaper) and the logistics. We have talked about a Contributor's copy though, versus a regular copy if any non-contributors wanted it. Though our priority if we go ahead with this will be to sort the contributors out first.

Also need to consider that many contributors contribute in multiple roles (e.g. Author in one chapter, Reviewer on another), which might be another reason to just go for some kind of "Contributor" badging.

**Reply by NishuGoel** [reply]
I meant the words "Author/Reviewer/Analyst..." . But that'd still add to the cost as the roles too are at least 7-8 resulting in that many lots.
Agree with the multiple role bit completely!
